server/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "utseus_api",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "engines": {
+    "node": ">=4.0.0"
+  },
+  "dependencies": {
+    "cors": "^2.8.4",
+    "debug": "^3.1.0",
+    "ethereum-input-data-decoder": "0.0.3",
+    "ethereumjs-testrpc": "^6.0.3",
```

Comment:
isn't test-rpc and truffle devDependencies? 

as an aside, in a production system, what are the steps to deploy?

Reply:
Could you please give more details what do you mean by first question? 

Regarding second question - it is good point. I have to think about this use case. May be tomorrow would be right time investigate deploy onto production version of ethereum.